test/problems/brownden.jl

```diff
@@ -10,10 +10,10 @@ function brownden_autodiff()
 
   x0 = [25.0; 5.0; -5.0; -1.0]
   f(x) = begin
-    s = 0.0
+    T = eltype(x)
```

Comment:
x0 sill has eltype Float64 though, and that will make it into the model. Is that a problem?

Reply:
The default model is `Float64`. The `obj` called with a different `eltype` of `x` will be fine.